src/hubverse_infrastructure/hubs/hubs.yaml

```diff
@@ -8,5 +8,6 @@ hubs:
 - hub: bsweger-flusight-forecast
```

Comment:
Will remove last week's test bucket in a follow-up PR (the bucket delete will fail anyway, because there's data in it)